users/templates/users/profile.html

```diff
@@ -32,6 +32,23 @@ <h2>{{ author.user.username }}</h2>
 
 </div>
 <div class="profile-body">
+
+  <div id="editProfileForm" class="edit-profile">
+    <div class="edit-profile-content">
```

Comment:
why the double div?

Reply:
Intentional. The outer div is what dims the background when the popup form is open and is also what causes the form to close if you click outside the form.
The inner div is the actual form